web/src/org/purefn/kurosawa/web/instrument.clj

```diff
@@ -0,0 +1,115 @@
+(ns org.purefn.kurosawa.web.instrument
+  "Thread-pool instrumentation for Ring-compliant web server. In most cases you may want
+   to use only `initialize` and `instrument` and use the output to launch the web server.
+
+   This library does not include prometheus (iapetos and bidi) and preflex as dependencies
```

Comment:
Maybe we should just consider this library "batteries-included" and include, iapetos, bidi and preflex as dependencies in in the web `project.clj`, what do you think?

Reply:
We can consider Kurosawa batteries-included - would it make sense to pursue this change in a future release for the following reasons?
- Keep this release small (causing least change to `services`)
- Future removal of Immutant may be clubbed together with the dependencies rehash
- Impact of future OpenAPI/Swagger changes on Bidi